critiquebrainz/frontend/external/musicbrainz_db/serialize.py

```diff
@@ -83,19 +83,56 @@ def to_dict_places(place, includes={}):
     return data
 
 
+def to_dict_artist_credit_names(artist_credit_name):
+    data = {
+        'name': artist_credit_name.name,
+        'artist': to_dict_artists(artist_credit_name.artist),
+    }
+    if artist_credit_name.join_phrase:
+        data['join_phrase'] = artist_credit_name.join_phrase
+    return data
+
+
 def to_dict_release_groups(release_group, includes={}):
```

Comment:
Thanks. Regarding the test database, we can make use ot `mbdata.create_sample_data()`. I've started some work here > https://github.com/ferbncode/critiquebrainz/commit/a27236eebacf8fd83efbcab9c993e3ee47837582 (works, but still WIP). I can make a list of testable entities (entities in [sample_data.py](https://bitbucket.org/lalinsky/mbdata/src/fd6a391d3d1c0a3085904ae65a50129a34cd1a51/mbdata/sample_data.py?at=master))

Also, these test (may) fail in case of schema_changes (in musicbrainz database). But I think that would be good for us?